src/components/structures/auth/AuthHeaderContext.tsx

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
https://github.com/vector-im/element-meta/pull/216 implores us to only have one one exported component per file, this has 3